pkg/moov/http.go

```diff
@@ -159,10 +159,19 @@ func (r *httpCallResponse) RequestId() string {
 	return ""
 }
 
+type errorResponse struct {
+	Error string `json:"error"`
+}
+
 func (r *httpCallResponse) Error() string {
 	generic := fmt.Sprintf("error from moov - status: %s http.request_id: %s http.status_code: %d", r.Status().Name, r.RequestId(), r.StatusCode())
 
 	switch r.StatusCode() {
+	case http.StatusBadRequest:
```

Comment:
Sadly this can be either this model where its just the `error` message or it can be the more complex validation error.

Reply:
Do we have examples of the other error types? I can adapt the ozzo reformatting to this if it's present.

Reply:
Looks like a bunch of services support this now - https://www.notion.so/moovfinancial/Core-Services-API-Responses-Update-ace9f9af54b444f8acfceb6a5b89bad7?pm=c

Reply:
With a 422 I can check for both error and ozzo responses and handle both.